go.mod

```diff
@@ -39,6 +39,7 @@ require (
 	github.com/mattn/go-colorable v0.1.12 // indirect
 	github.com/mattn/go-isatty v0.0.14 // indirect
 	github.com/mitchellh/mapstructure v1.4.3 // indirect
+	github.com/muja/goconfig v0.0.0-20180417074348-0a635507dddc
```

Comment:
this should go up in the first requires stanza as its it a direct dependency. However, I wonder:

1) Will github.com/pelletier/go-toml work? we already depend on that, so one less dep to vet.
2) Should we do this directly or use `git config` commands. Its a pretty advanced command so I worry we might mess something up. It also might let you drop some code that's trying to discover where gitconfig is located

Reply:
Removed the dep entirely and just used two commands.